app/containers/SongContainer/index.js

```diff
@@ -0,0 +1,250 @@
+import React, { useEffect, memo, useState } from 'react';
```

Comment:
formatting is off in a lot of places in this file, please make sure linting doesn't fail before pushing

Reply:
Will take care next time